client/components/UserMenu.tsx

```diff
@@ -41,6 +44,10 @@ export default function UserMenu() {
                 </DropdownMenuTrigger>
                 <DropdownMenuPortal>
                     <DropdownMenuContent>
+                        <DropdownMenuItem onClick={() => setAnnouncementsOpen(true)}>
+                            <Bell />
+                            Announcements
+                        </DropdownMenuItem>
```

Comment:
Perhaps move this out to its own button next to the user icon: 
![image](https://github.com/user-attachments/assets/f6ae7931-e014-4038-afdb-f23702e97a86)